.gitignore

```diff
@@ -17,6 +17,10 @@
 node_modules
 .idea
 
+# production
+apps/monk-test-app/build
+apps/monk-test-app-native/build
+
```

Comment:
Remove these lines from the gitignore at the root of the repo and create 2 gitignores inside the apps/monk-test-app and apps/monk-test-app-native directories where you add the build param

Reply:
Done